sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -390,6 +402,25 @@ void ExtrinsicReflectorBasedCalibrator::trackingRequestCallback(
   RCLCPP_INFO(this->get_logger(), "New converged detections: %d", current_new_tracks_);
 }
 
+void ExtrinsicReflectorBasedCalibrator::deleteTrackRequestCallback(
+  __attribute__((unused)) const std::shared_ptr<std_srvs::srv::Empty::Request> request,
+  __attribute__((unused)) const std::shared_ptr<std_srvs::srv::Empty::Response> response)
+{
+  using std::chrono_literals::operator""s;
+
+  if (converged_tracks_.size() > 0) {
+    converged_tracks_.pop_back();
+    calibrateSensors();
+    visualizeTrackMarkers();
+    drawCalibrationStatusText();
+    RCLCPP_INFO(
+      this->get_logger(), "You delete one previous track, there are %d converged tracks remain",
```

Comment:
You delete one previous track -> The last track was successfully deleted. Remaining converged tracks: %d

Reply:
@vividf
When you push a commit addressing a comment, it is customary to mention the reviewer in the comment and pasting the hash of the commit that addresses the comment (github should convert it into a easily readable link)